app/models/true_work_income_validation.rb

```diff
@@ -0,0 +1,113 @@
+require 'truework'
+require 'base64'
+
+class TrueWorkIncomeValidation < IncomeValidation

```

Comment:
It turns out we're not validating income at all.  We're querying or reporting income.  So maybe this class, and superclasses, should be renamed.